### PR TITLE
Poppler 0.69.0

### PR DESCRIPTION
**Pull request by edporras**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by lembacon** [comment]
Build failure regarding `pdftoipe`,

```
09:45:33 ==> brew install --verbose --build-bottle pdftoipe
09:45:41 ==> FAILED
09:45:41 /usr/bin/sandbox-exec -f /private/tmp/homebrew20180924-96346-1hrvl33.sb nice /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/../ruby/2.3.0/gems/ruby-macho-2.0.0/lib:/usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/../ruby/2.3.0/gems/plist-3.4.0/lib:/usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/:/usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/../ruby/2.3.0/gems/backports-3.11.4/lib:/usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/../ruby/2.3.0/gems/activesupport-5.2.1/lib:/usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/../ruby/2.3.0/gems/tzinfo-1.2.5/lib:/usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/../ruby/2.3.0/gems/thread_safe-0.3.6/lib:/usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/../ruby/2.3.0/gems/minitest-5.11.3/lib:/usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/../ruby/2.3.0/gems/i18n-1.1.0/lib:/usr/local/Homebrew/Library/Homebrew/vendor/bundle-standalone/bundler/../ruby/2.3.0/gems/concurrent-ruby-1.0.5/lib:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/gems/2.3.0/gems/did_you_mean-1.0.0/lib:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/site_ruby/2.3.0:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/site_ruby/2.3.0/x86_64-darwin9.0:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/site_ruby/2.3.0/universal-darwin9.0:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/site_ruby:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/vendor_ruby/2.3.0:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/vendor_ruby/2.3.0/x86_64-darwin9.0:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/vendor_ruby/2.3.0/universal-darwin9.0:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/vendor_ruby:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/2.3.0:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/2.3.0/x86_64-darwin9.0:/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.3.7/lib/ruby/2.3.0/universal-darwin9.0:/usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/build.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/pdftoipe.rb --build-bottle --verbose
09:45:41 ==> Downloading https://github.com/otfried/ipe-tools/archive/v7.2.7.tar.gz
09:45:41 Already downloaded: /Users/brew/Library/Caches/Homebrew/downloads/5b8f05bf643f5cf2099afb61132307dbf311629c2cbf7eeddae0c24b710ce1f5--ipe-tools-7.2.7.tar.gz
09:45:41 ==> Verifying 5b8f05bf643f5cf2099afb61132307dbf311629c2cbf7eeddae0c24b710ce1f5--ipe-tools-7.2.7.tar.gz checksum
09:45:41 tar xf /Users/brew/Library/Caches/Homebrew/downloads/5b8f05bf643f5cf2099afb61132307dbf311629c2cbf7eeddae0c24b710ce1f5--ipe-tools-7.2.7.tar.gz -C /private/tmp/d20180924-96348-dlqy6h
09:45:41 cp -pR /private/tmp/d20180924-96348-dlqy6h/ipe-tools-7.2.7/. /private/tmp/pdftoipe-20180924-96348-5w0hgn/ipe-tools-7.2.7
09:45:41 ==> make
09:45:41 clang++ -Wno-write-strings -I/usr/local/Cellar/poppler/0.69.0/include/poppler  -c -o parseargs.o parseargs.cc
09:45:41 clang++ -Wno-write-strings -I/usr/local/Cellar/poppler/0.69.0/include/poppler  -c -o xmloutputdev.o xmloutputdev.cpp
09:45:41 clang++ -Wno-write-strings -I/usr/local/Cellar/poppler/0.69.0/include/poppler  -c -o pdftoipe.o pdftoipe.cpp
09:45:41 pdftoipe.cpp:160:11: error: no type named 'memCheck' in 'Object'
09:45:41   Object::memCheck(stderr);
09:45:41   ~~~~~~~~^
09:45:41 1 error generated.
09:45:41 make: *** [pdftoipe.o] Error 1
09:45:41 make: *** Waiting for unfinished jobs....
09:45:41 
```

File: Formula/poppler.rb [hunk 1]
```diff
@@ -4,15 +4,8 @@ class Poppler < Formula
...
-    url "https://poppler.freedesktop.org/poppler-0.68.0.tar.xz"
-    sha256 "f90d04f0fb8df6923ecb0f106ae866cf9f8761bb537ddac64dfb5322763d0e58"
-
-    # https://gitlab.freedesktop.org/poppler/poppler/merge_requests/18
-    # Should be safe to remove on next release but check if merged.
-    patch do
-      url "https://gitlab.freedesktop.org/poppler/poppler/merge_requests/18.diff"
-      sha256 "ef7d7b7fed6799911b7ef925395b8de552aadaf3754d7b02fe3505649518808c"
-    end
+    url "https://poppler.freedesktop.org/poppler-0.69.0.tar.xz"
```
Comment (commitay) [inline Formula/poppler.rb]: The `stable do` block isn't needed, the `url` and `sha256` can be moved above `head`.

**Comment by fxcoudert** [comment]
Reported the `pdftoipe` failure to compile: https://github.com/otfried/ipe-tools/issues/29

**Comment by fxcoudert** [comment]
`pdftoipe` can be updated at the same time, which will fix the issue: 
https://github.com/otfried/ipe-tools/archive/v7.2.7.1.tar.gz

**Comment by edporras** [comment]
Ok, pushed updated formula for new `pdftoipe` version

**Pull request closed without merging**